### PR TITLE
fixed 5 canvas2d bugs.

### PR DESCRIPTION
**Pull request by wyklion**

### CODE REVIEW

File: canvas2D/src/Engine/babylon.group2d.ts [hunk 1]
```diff
@@ -373,12 +373,9 @@
 
             // The dimension must be overridden when using the designSize feature, the ratio is maintain to compute a uniform scale, which is mandatory but if the designSize's ratio is different from the rendering surface's ratio, content will be clipped in some cases.
             // So we set the width/height to the rendering's one because that's what we want for the viewport!
-            if (this instanceof Canvas2D) {
-                let c = <Canvas2D><any>this;
-                if (c.designSize != null) {
-                    sw = this.owner.engine.getRenderWidth();
-                    sh = this.owner.engine.getRenderHeight();
-                }
+            if ((this instanceof Canvas2D || this.id === "__cachedCanvasGroup__") && this.owner.designSize != null) {
```
Comment (nockawa) [inline canvas2D/src/Engine/babylon.group2d.ts]: I'm not sure this one will work in every cases, but I'll merge it anyway and make tests

File: canvas2D/src/Engine/babylon.group2d.ts [hunk 2]
```diff
@@ -912,23 +914,12 @@
             // For now we only support these property changes
             // TODO: add more! :)
             switch (prop.id) {
-                case Prim2DBase.actualScaleProperty.id:
                 case Prim2DBase.actualPositionProperty.id:
```
Comment (nockawa) [inline canvas2D/src/Engine/babylon.group2d.ts]: can you explain why you suppress this actual block of code please?

File: dist/preview release/babylon.canvas2d.d.ts [hunk 3]
```diff
@@ -406,6 +406,7 @@ declare module BABYLON {
          * When enabled the Prim2DBase.pointerEventObservable property will notified when appropriate events occur
          */
         interactionEnabled: boolean;
+        fitRenderingDevice: boolean;
```
Comment (nockawa) [inline dist/preview release/babylon.canvas2d.d.ts]: you were not supposed to add the d.ts I think it was a mistake but that's fine this time the impact is not negative

File: canvas2D/src/Engine/babylon.group2d.ts (hunk 2 above)
Comment (wyklion) [inline canvas2D/src/Engine/babylon.group2d.ts]: I think the position need not multiple scale. 
like i modify in _allocateGroupCache too.

Comment (nockawa) [inline canvas2D/src/Engine/babylon.group2d.ts]: ok, I merge

File: dist/preview release/babylon.canvas2d.d.ts (hunk 3 above)
Comment (wyklion) [inline dist/preview release/babylon.canvas2d.d.ts]: ok... i'm not very clear about this.

**Pull request merged**